src/styles/components/collapsible-error-message.less

```diff
@@ -0,0 +1,54 @@
+
+//
+// The CollapsibleErrorMessage is a component used on top
+// of forms or on other places to display a main error
+// message and a floating drop-down with the error details
+//
+.collapsible-error-message {
+
+  //
+  // Shared styling between the two items
+  //
+  .collapsible-fixed, .collapsible-floating > div {
+    background: color-lighten(@red, 90);
+    padding: (@base-spacing-unit / 2) @base-spacing-unit;
+  }
+
+  //
+  // The fixed part of the error message
+  //
+  .collapsible-fixed {
+      border-bottom: @red 1px solid;
+  }
+
+  //
+  // The floating part of the error message
+  //
+  .collapsible-floating {
+    position: relative;
+
+    > div {
+
+      // Fill the fixed part dimentions
+      position: absolute;
+      left: 0; width: 100%;
+
+      // Bring on fore-most z-index
+      z-index: 100000;
```

Comment:
Ok, this is up for discussion. Since this is supposed to be re-usable it can be used either in a modal or on the main page body. But if it's used in the page body *and* a modal is shown on top, the floater would be visible on top of the modal.

I guess a property `modal=true` can be useful in order to select a `z-index` above the modal or below? Is there any better solution?